contract/gql_mutations/delete_contract.py

```diff
@@ -0,0 +1,43 @@
+from functools import lru_cache
+
+from contract.gql_mutations import ContractInputType, ContractNotExistException
+from contract.models import Contract
+from core.schema import OpenIMISMutation
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import ValidationError, PermissionDenied
+
+
+class DeleteContractMutation(OpenIMISMutation):
+    _mutation_class = "DeleteContractMutation"
+
+    class Input(ContractInputType):
+        pass
+
+    @classmethod
+    def async_mutate(cls, user, **data):
+        output = []
+        for contract_uuid in data["uuids"]:
+            deletion_result = super(DeleteContractMutation, cls).async_mutate(user, contract_uuid=contract_uuid)
```

Comment:
I might have a flaw in my logic (using history) for deleted object, we might need to add a member isActive/IsDeleted to UUIDHistoryModel so this flag is changed when a row is "Deleted"